thriftbp/client_middlewares.go

```diff
@@ -85,10 +85,10 @@ type DefaultClientMiddlewareArgs struct {
 	// If it's not set, the global one from ecinterface.Get will be used instead.
 	EdgeContextImpl ecinterface.Interface
 
-	// The value to be set on the User-Agent THeader.
+	// The value to be set on the "User-Agent" (HeaderUserAgent) THeader.
```

Comment:
I would say something more like "The value that the server will receive as the PeerName for connections initiated by this client." or something.

Whigh might mean that we want a name like "OriginPeer" or "SendPeerService" or something?

Reply:
*PeerService

Reply:
how about something like "The name for the server to identify this client, via the "User-Agent" (HeaderUserAgent) THeader."?